sdk/test-utils/perfstress/GettingStarted.md

```diff
@@ -0,0 +1,263 @@
+  - [Track 2](#Setting-up-the-project)
+  - [Track 1](#For-perf-testing-track-1-version-of-the-same-package)
+- [Writing perf tests](#writing-perf-tests)
+  - [Entry Point](#Entry-point)
+  - [Base Class](#Base-Class)
+  - [Test File](#Test-file)
+  - [Custom Options](#custom-options)
+- [Executing the perf tests](#executing-the-perf-tests)
+  - [Command to run](#Command-to-run)
+  - [Adding Readme/Instructions](#Adding-Readme/Instructions)
+  - [Testing an older track 2 version](#Testing-an-older-track-2-version)
+
+## [Setting up the project](#Setting-up-the-project)
+
+To add perf tests for the `sdk/<service>/<service-sdk>` package, follow the steps below.
+
+1.  Create a new folder for the perf tests.
+
+    Path- `sdk/<service>/perf-tests/<service-sdk>`
+
+    (Create the `perf-tests` folder if that doesn't exist)
+
+2.  This new perf test project will be managed by the rush infrastructure in the repository, with the package name `@azure-tests/<service-sdk>`. To allow rush to manage the project, add the following entry in `rush.json`
+
+    ```
+        {
+          "packageName": "@azure-tests/perf-<service-sdk>",
+          "projectFolder": "sdk/<service>/perf-tests/<service-sdk>",
+          "versionPolicyName": "test"
+        }
+
+    ```
+
+3.  Tests will live under `sdk/<service>/perf-tests/<service-sdk>/test`
+4.  Add a `package.json` such as [example-perf-package.json](https://github.com/Azure/azure-sdk-for-js/blob/fe9b1e5a50946f53b6491d7f67b2420d8ee1b229/sdk/storage/perf-tests/storage-file-datalake/package.json) at `sdk/<service>/perf-tests/<service-sdk>` folder.
+
+    Make sure to import your `<service-sdk>` and the `test-utils-perfstress` project.
+
+    ```json
+      "dependencies": {
+         "@azure/<service-sdk>": "^<version-in-master-branch>",
+         "@azure/test-utils-perfstress": "^1.0.0"
+       }
+    ```
+
+    _Note: `"@azure/test-utils-perfstress"` is not a published npm package._
+
+    Set the name of the package and mark it as private.
+
+    ```json
+     "name": "@azure-tests/perf-<service-sdk>",
+     "private": true,
+    ```
+
+5.  Run `rush update` and commit the changes to the `pnpm-lock` file.
+6.  Copy the `tsconfig.json`, `sample.env`(and `.env`) files that are present at the `sdk/<service>/<service-sdk>` to `sdk/<service>/perf-tests/<service-sdk>`.
+
+    TSCONFIG
+
+    - Modify the "extends" string in the copied tsconfig by adding ".." since the perf tests project is located a level below the actual SDK.
+    - Set the `compilerOptions.module` to `commonjs` in the `tsconfig` to allow running the tests with `ts-node`.
+
+    In the end, your tsconfig may look something like below.
+
+    ```
+        {
+          "extends": "../../../../tsconfig.package",
+          "compilerOptions": {
+            "module": "CommonJS",
+            "target": "ES2015",
+            "declarationDir": "./typings/latest",
+            "lib": ["ES6", "ESNext.AsyncIterable", "DOM"],
```

Comment:
Adding DOM was because of an issue in core v2 that is about to be fixed so it should not be needed if the library has shimmed the types it needs. cc @witemple-msft
```suggestion
            "lib": ["ES6", "ESNext.AsyncIterable"],
```